CHANGELOG.md

```diff
@@ -11,6 +11,16 @@ this platform. FreeBSD builds will return in a future release.
 - [FEATURE] New integration:
   [postgres_exporter](https://github.com/wrouesnel/postgres_exporter) (@rfratto)
 
+- [ENHANCEMENT] The Agent can now send traces to a compatible backend. To do
+  configure sending spans from the Agent, the environment variable
```

Comment:
Do you mean spans from the agent itself or a system it observes?

Reply:
From the Agent itself, let me clarify 👍